src/modules/shortcut_guide/shortcut_guide.h

```diff
@@ -68,4 +68,16 @@ class OverlayWindow : public PowertoyModuleIface
             { L"dark", IDS_SETTING_DESCRIPTION_THEME_DARK }
         };
     } theme;
+
+    struct DisplayOnMethod
+    {
+        PCWSTR name = L"display on method";
+        std::wstring value = L"monitor";
+        int resourceId = IDS_SETTING_DISPLAY_ON;
+        std::vector<std::pair<std::wstring, UINT>> keys_and_texts = {
+            { L"main display", IDS_SETTING_DISPLAY_ON_MAIN_DISPLAY},

```

Comment:
Since we use this string in `overlay_window.cpp`, it would be better to make them constant.